uat/custom-components/client-java-paho/src/main/java/com/aws/greengrass/testing/mqtt5/client/MqttConnection.java

```diff
@@ -14,6 +14,7 @@
 import lombok.NonNull;
 
 import java.util.List;
+import java.util.Map;
```

Comment:
user properties is NOT a map. It is a list of keys and values. Duplicate keys are allowed.

Reply:
Looks so:

```3.1.2.11.8 User Property
38 (0x26) Byte, Identifier of the User Property.

Followed by a UTF-8 String Pair.

 

The User Property is allowed to appear multiple times to represent multiple name, value pairs. The same name is allowed to appear more than once.
```
Please read the[ MQTT v5.0 spec](https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html#_Toc3901054)

That means in proto file `map<string, string> userProperties` could not be used.

Possible solution:
```
message Mqtt5Properties {
    string name = 1;
    string value = 2;
};
```
and 
```
repeated Mqtt5Properties properties = 5;
```
and so one.